### ISSUE TITLE
Bug: formatting pug with tag interpolation

### ISSUE THREAD

**Issue by Christilut** [vuejs/vetur#2282]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: Win
- Vetur version: 0.27.3
- VS Code version: 1.49

## Problem

When formatting pug code, it seems that [tag interpolation](https://pugjs.org/language/interpolation.html) e.g. `#[ ]` is treated incorrectly.

```pug
p Click #[a.link(href='https://www.example.com' rel='noreferrer' target='_blank') here] to go somewhere
```

Turns into:

```pug
    p Click #[a.link(
      href='https://www.example.com'
      rel='noreferrer'
      target='_blank') here] to go somewhere
```

This gives a webpack build error:

```
  > 23|     p Click #[a.link(
-----------------------------^
    24|       href='https://www.example.com'
    25|       rel='noreferrer'
    26|       target='_blank') here] to go somewhere

The end of the string reached with no closing bracket ) found.
```

I think everything between `#[ ]` should remain on the same line.

**Comment by Shinigami92** [comment]
Referencing https://github.com/prettier/plugin-pug/issues/107

I created a test for this https://github.com/prettier/plugin-pug/pull/108 and it turns out that my plugin doesn't do this on its own...
So we think it's a problem with vetur 🤷‍♂️

**Comment by octref** [comment]
It's Vetur's issue. I'll fix it.

**Comment by octref** [comment]
Fixed #2014 and updated plugin-pug dependency.
